examples/fc_networks.py

```diff
@@ -63,6 +63,11 @@
 except HPOneViewException as e:
     print(e.msg['message'])
 
+# Get by Uri
```

Comment:
Thinking that user could run full sample file, so we had used try block on 'Get by Id' example because that is a fake id (It might not exist on OneView), so is expected  fail.
On the other hand the 'Get by uri' won't fail because we pass as argument the uri from the previously created resource